deploy/csi-azurefile-driver.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: file.csi.azure.com
+spec:
+  attachRequired: true
+  podInfoOnMount: true
+  volumeLifecycleModes:  # added in Kubernetes 1.16
```

Comment:
pls remove this, there are users are still using 1.16, we don't want to break those users.

Reply:
done